apps/pristine/src/components/message.tsx

```
@@ -0,0 +1,241 @@
+                      <div
+                        className={cn("flex flex-col gap-4", {
+                          "bg-secondary text-secondary-foreground px-3 py-2 rounded-tl-xl rounded-tr-xl rounded-bl-xl":
+                            message.role === "user",
+                        })}
+                      >
+                        <Markdown>{part.text}</Markdown>
+                      </div>
+                    </motion.div>
+                  );
+                case "tool-invocation":
+                  const { toolName, state } = part.toolInvocation;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)